src/Menu/CachedBackendMenuBuilder.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Menu;
+
+use Psr\SimpleCache\CacheInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class CachedBackendMenuBuilder
```

Comment:
implements BackendMenuBuilderInterface

Reply:
Why? That's making it more abstract and fuzzy, imho, and lead to a path we went down with on v3, where we make things more complex than needed.

Reply:
Because then you can disable cache, stopwatch etc. just by editing config file.